doc/rados/operations/erasure-code.rst

```diff
@@ -61,6 +61,31 @@ the following profile can be defined::
     $ rados --pool ecpool get NYAN -
     ABCDEFGHI
 
+Formula for calculating the raw size percentage overhead::
+
+    K/(K+M) * 100
+
+For example for profile (*K=3*) and (*M=2*) the percentage overhead is 60%.
+
+Formula for calculating the raw size overhead::
```

Comment:
Using the term "overhead" here is incorrect. As @jdurgin said, this ratio helps calculate the "effective space available" given the raw space.